apache/uima-ruta
# [UIMA-6443] Fresh Eclipse update site for every release

**JIRA Ticket:** https://issues.apache.org/jira/browse/UIMA-6443

**What's in the PR**
- UIMA Parent POM 14 -> 15-SNAPSHOT
- Remove jave varsion enforcer configurations which are present already in the parent pom
- Update jiraVersion property
- Add Eclipse feature and update site to the regular build
- Remove changes plugin config and trigger it via a marker file instead
- Convert feature and update site to using tycho
- Activate auto-staging
- Fold properties into the feature.xml because Tycho does not seem to support externalized strings

**How to test manually**
* Perform a (fake) release build
* Check if the issue report is correctly generated
* Initialize a local svn repo: `svnadmin create /my/local/testrepo`
* Do a test build: `mvn -Papache-release -DskipTests -Dmaven.deploy.skip -Dstaging-scm-root='scm:svn:file:///my/local/testrepo/' clean deploy`
* Check if the commit made it in `svn log file:///my/local/testrepo/`
* Check if Eclipse plugins can be installed from generated site

**Automatic testing**

**Documentation**

**Organizational**
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
